app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    phrase_lower = phrase.lower()
+    letter_lower = letter.lower()
+
+    occurrences = phrase_lower.count(letter_lower)
+
+    return occurrences
```

Comment:
It can be done in a few simpler without creation of variables
```suggestion
    return phrase.lower().count(letter.lower())
```